src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -437,5 +442,34 @@ internal void ScriptBackup(BackupOperation backupOperation)
         {
             backupOperation.Execute(TaskExecutionMode.Script);
         }
+
+        internal async Task HandleCreateSasRequest(
```

Comment:
Please make a new service for this - it doesn't really seem to belong to "disaster recovery"